app/controllers/application_controller.rb

```diff
@@ -65,37 +102,18 @@ def is_staging
       ['staging'].include?(Rails.env)
     end
 
-    ##
-    # Extra authentication for palni-palci during development phase
-    def authenticate_if_needed
```

Comment:
There remains a `before_action :authenticate_if_needed` declaration on line 21.  Yet we're removing this method.  Which should it be?  My understanding is that without this in place, the staging environment will no longer require HTTP Auth.